### ISSUE TITLE
Error On starting elm-instant (after everything updated)

### ISSUE THREAD

**Issue by desmond-dsouza** [mukeshsoni/elm-instant#11]
I get the error as soon as try to start elm-instant ... in case the error report is helpful:

**Atom Version**: 1.11.1
**Electron Version**: 0.37.8
**System**: Mac OS X 10.11.6
**Thrown From**: [elm-instant](https://github.com/mukeshsoni/elm-instant) package, v0.5.1
### Stack Trace

Uncaught TypeError: Arguments to CompositeDisposable.add must have a .dispose() method

```
At /Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/composite-disposable.js:70

TypeError: Arguments to CompositeDisposable.add must have a .dispose() method
    at assertDisposable (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/composite-disposable.js:70:13)
    at CompositeDisposable.module.exports.CompositeDisposable.add (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/composite-disposable.js:43:11)
    at new CompositeDisposable (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/composite-disposable.js:19:14)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:265:29
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at PaneContainer.module.exports.PaneContainer.addedPaneItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane-container.js:388:27)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/pane-container.js:366:19
    at /Applications/Atom.app/Contents/Resources/app.asar/src/pane-container.js:97:16
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at PaneContainer.module.exports.PaneContainer.didAddPane (/Applications/Atom.app/Contents/Resources/app.asar/src/pane-container.js:306:27)
    at Pane.module.exports.Pane.setContainer (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:131:26)
    at PaneAxis.module.exports.PaneAxis.addChild (/Applications/Atom.app/Contents/Resources/app.asar/src/pane-axis.js:144:13)
    at PaneAxis.module.exports.PaneAxis.insertChildAfter (/Applications/Atom.app/Contents/Resources/app.asar/src/pane-axis.js:216:19)
    at Pane.module.exports.Pane.split (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:923:23)
    at Pane.module.exports.Pane.splitRight (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:884:19)
    at Object.createPlaygroundPane (/Users/desmond/.atom/packages/elm-instant/lib/elm-instant.js:110:42)
    at Object.toggle (/Users/desmond/.atom/packages/elm-instant/lib/elm-instant.js:170:12)
    at atom-workspace.elmInstantToggle (/Users/desmond/.atom/packages/elm-instant/lib/elm-instant.js:43:40)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/command-palette/lib/command-palette-view.js:183:32)
    at CommandPaletteView.module.exports.SelectListView.confirmSelection (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
    at space-pen-div.atom.commands.add.core:confirm (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:114:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```
### Commands

```
  4x -0:11.9.0 blur (atom-text-editor.editor.is-focused)
     -0:03.2.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:02.9.0 blur (atom-text-editor.editor.is-focused)
     -0:00.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:00.4.0 blur (atom-text-editor.editor.mini.is-focused)
     -0:00.1.0 elm-instant:toggle (atom-text-editor.editor)
```
### Config

``` json
{
  "core": {}
}
```
### Installed Packages

``` coffee
# User
atom-elixir, v0.2.1 (active)
autocomplete-go, v1.3.0 (active)
builder-go, v1.0.1 (active)
elm-format, v2.1.0 (active)
elm-instant, v0.5.1 (inactive)
go-config, v1.2.4 (active)
go-debug, v0.1.22 (active)
go-get, v2.1.1 (active)
go-plus, v4.3.2 (active)
godoc, v1.1.1 (active)
gofmt, v1.2.0 (active)
gometalinter-linter, v1.2.2 (active)
gorename, v1.1.0 (active)
haskell-ghc-mod, v1.18.0 (inactive)
ide-haskell, v1.8.1 (inactive)
ide-haskell-repl, v0.1.10 (inactive)
language-elixir, v0.16.0 (active)
language-elm, v1.5.0 (active)
language-haskell, v1.9.9 (active)
linter, v1.11.18 (active)
linter-elm-make, v0.17.7 (active)
navigator-go, v1.1.3 (active)
Remote-FTP, v0.9.0 (active)
term3, v0.22.1 (active)
tester-go, v2.0.2 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.44.0 (inactive)
base16-tomorrow-dark-theme, v1.2.0 (inactive)
base16-tomorrow-light-theme, v1.2.0 (inactive)
one-dark-ui, v1.6.0 (active)
one-light-ui, v1.6.0 (inactive)
one-dark-syntax, v1.3.0 (active)
one-light-syntax, v1.3.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.7.0 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.2 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.42.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (active)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.40.0 (active)
find-and-replace, v0.201.1 (inactive)
fuzzy-finder, v1.4.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.2 (active)
image-view, v0.59.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v1.0.0 (active)
notifications, v0.65.1 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.242.2 (active)
snippets, v1.0.2 (active)
spell-check, v0.68.2 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.0 (active)
symbols-view, v0.113.1 (inactive)
tabs, v0.101.0 (active)
timecop, v0.33.2 (active)
tree-view, v0.209.3 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.35.1 (active)
whitespace, v0.33.0 (active)
wrap-guide, v0.38.2 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.2 (active)
language-csharp, v0.12.1 (active)
language-css, v0.37.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.42.1 (active)
language-html, v0.45.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.23.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.2 (active)
language-less, v0.29.5 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.2 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.69.0 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.56.0 (active)
language-shellscript, v0.22.4 (active)
language-source, v0.9.0 (active)
language-sql, v0.23.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.9 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by mukeshsoni** [comment]
@desmond-dsouza Thanks for reporting. Might be a problem with new version of atom. Will check it out.

**Comment by mukeshsoni** [comment]
yeah, looks like many packages which depend on split pane broke after atom 1.11.1 release. The issue is being tracked here - https://github.com/atom/atom/issues/12963

I don't know what a good solution for elm-instant is.

**Comment by mukeshsoni** [comment]
Hey @desmond-dsouza , the issue is fixed in 1.11.2 - https://github.com/atom/atom/issues/12963
